src/broadcast/data.rs

```diff
@@ -6,6 +6,8 @@
 // License, Version 2.0 found in the LICENSE-APACHE file in the root directory
 // of this source tree.
 
+use std::fmt::Debug;
+
```

Comment:
Ah, thanks. I was wondering why fmt wasn't automatically alphabetizing these, but I guess it's because it only alphabetizes in "chunks" that aren't newline-separated.